package Prectice_Exception;

public class AllInOneException {

	public static void main(String[] args) {
		int a=5;
		int b=0;
		int c;
		String name="ram";
		String s=null;
//		c=a/b;
//		System.out.println(c);
		int arr[]= {1,2,3};
		try {
			c=a/b;
			System.out.println(c);
		} catch ( ArithmeticException e) {
			System.out.println(e);
		}
		try {
			System.out.println(name.charAt(5));
		} catch (StringIndexOutOfBoundsException e) {
			System.out.println(e);
		}
		try {
			System.out.println(s.charAt(3));
		} catch (Exception e) {
			System.out.println(e);
		}

	}

}
